src/sizes.rs

```diff
@@ -30,3 +27,30 @@ pub const THEORETICAL_MAX_MESSAGE_SIZE: usize = PACKET_SIZE - 7 + 128 * (PACKET_
 pub const LARGE_BLOB_MAX_FRAGMENT_LENGTH: usize = 0;
 #[cfg(feature = "large-blobs")]
 pub const LARGE_BLOB_MAX_FRAGMENT_LENGTH: usize = 3008;
+
+// TODO: update these, and grab them from a common crate?
+cfg_if::cfg_if! {
+    if #[cfg(feature = "mldsa87")] {
+        pub const MAX_MESSAGE_LENGTH: usize = MAX_COMMITTMENT_LENGTH;
+        pub const MAX_CREDENTIAL_ID_LENGTH: usize = 7523 + 57 + 30 + 37;
+        pub const AUTHENTICATOR_DATA_LENGTH: usize = MAX_CREDENTIAL_ID_LENGTH + 2031; // TODO: this will have to be larger
+        pub const ASN1_SIGNATURE_LENGTH: usize = 4627;
```

Comment:
Please add a comment that explains how the values are calculated.  Is the TODO comment still applicable?